esmvalcore/cmor/_fixes/cmip6/fio_esm_2_0.py

```diff
@@ -0,0 +1,64 @@
+"""Fixes for FIO-ESM-2-0 model."""
+import iris
+
+from ..common import OceanFixGrid
+from ..fix import Fix
+from ..shared import round_coordinates
+
+Tos = OceanFixGrid
+
+
+class Omon(Fix):
+    """Fixes for Omon vars."""
+
+    def fix_metadata(self, cubes):
+        """Fix latitude and longitude with round to 6 d.p.
+
+        Parameters
+        ----------
+        cubes : iris.cube.CubeList
+            Input cubes.
+
+        Returns
+        -------
+        iris.cube.CubeList
+        """
+        round_coordinates(cubes,
+                          decimals=6,
+                          coord_names=["longitude", "latitude"])
+        return cubes
+
+
+class Amon(Fix):
+    """Fixes for Amon vars."""
+
+    def fix_metadata(self, cubes):
+        """Fix metadata.
+
+        FIO-ESM-2-0 ta data contains error in coordinate bounds.
```

Comment:
```suggestion
        FIO-ESM-2-0 Amon data contains error in coordinate bounds.
```
I guess this is for the whole realm?